integrations/lando-pantheon/app.js

```diff
@@ -36,9 +36,11 @@ module.exports = (app, lando) => {
       });
     });
 
-    // Add tokens and other meta to our app
-    app.pantheonTokenCache = 'pantheon.tokens';
-    app.pantheonTokens = lando.cache.get(app.pantheonTokenCache) || [];
-    app.terminusTokens = utils.getTerminusTokens(lando.config.home);
+    // Load in other stuff like tokens and other meta at the most opportune moment
+    app.events.on('pre-init', 1, () => {
+      app.pantheonTokenCache = 'pantheon.tokens';
+      app.pantheonTokens = lando.cache.get(app.pantheonTokenCache) || [];
+      app.terminusTokens = utils.getTerminusTokens(lando.config.home);
+    });
```

Comment:
@dustinleblanc this was actually the big roadblock. The relevant `pull|push|switch` commands were using the stuff populated in these vars and these were all loading BEFORE the stuff we wanted to do. Wrapping them so they only load during an app.init event was the money move.